ui/app/styles/_page.scss

```diff
@@ -675,20 +680,22 @@
       box-shadow: 0 0 0 1px color.$black;
       border-radius: 2px 2px 0 0;
       background: url(/images/pattern.svg) no-repeat top center color.$black;
+      font-size: scale.$lg-14;
 
       @media (prefers-color-scheme: dark) {
         background-color: rgba(color.$black, 0.5);
         box-shadow: 0 0 0 1px rgb(var(--outline));
       }
 
       .icon.brand {
-        width: scale.$lg-14;
-        height: scale.$lg-14;
-
         path {
           fill: color.$black;
           stroke: $brand-color;
         }
+
+        > .pds-icon__svgWrapper {
+          display:flex;
+        }
```

Comment:
If there’s time, we could file an issue on the structure repo explaining why we needed to do this and suggest some ways to fix it at-source. It’s not a bug exactly but it would be better if we didn’t have to reach into the component like this.

Reply:
I just checked and it looks like some change I made later on fixed the need to add this. I removed this styling and we don't have to file the issue to the structure repo! 🙂